src/components/Button.js

```diff
@@ -41,15 +43,13 @@ module.exports = React.createClass({
 		};
 	},
 	render() {
+		var classes = this.props.sheet.classes;
```

Comment:
Some links regarding this topic:

https://gist.github.com/aldendaniels/5d94ecdbff89295f4cd6
https://gist.github.com/sebmarkbage/ef0bf1f338a7182b6775